patch/cisco-Enable-static-memory-reservation-for-OIRable-PCIe-de.patch

```diff
@@ -0,0 +1,198 @@
+commit 9a57f4339767fb8129ee54e9d1714af59d4a6937
+Author: balsup <balsup@contoso.com>
```

Comment:
Where is that patch taken from?

Please add the full name of the author.

Reply:
Please update the address to *Billie Alsup <balsup@cisco.com>*.

Reply:
Recreated patch